test/tests/commonjs-requires-in-comment.js

```diff
@@ -0,0 +1,15 @@
+"format cjs";
```

Comment:
This isn't a valid CommonJS file.

Try just writing:

```javascript
// This comment triggered SystemJS to do a require because of this -> require('')
exports.exportName = 'export';
```